src/psyclone/tests/nemo/transformations/openmp/openmp_test.py

```diff
@@ -201,6 +218,7 @@ def test_omp_do_update():
             "this Node has 2 children:" in str(err))
 
 
+@pytest.mark.xfail(reason="Can this test be removed?")
```

Comment:
I think so. The InternalError raised is slightly different now and it happens inside `_add_region`. Codecov shows `_add_region` is completely covered but I understand this comes from the OpenACC tests `test_add_region*`.

Maybe it would be good to make some isolated tests in `test/psyGen_test.py` of the `_add_region` method instead than in the OpenAcc/OpenMP transformations?

Reply:
Oops - that was supposed to be a note to myself to remind me to check! I'll try restructuring some tests as you suggest.

Reply:
Actually, on second thoughts, I've deleted these two tests but propose to leave the OpenACC ones as they are as these tests can be removed altogether as part of #435. Is that OK?